internal/env/auth_secret.go

```diff
@@ -0,0 +1,18 @@
+package env
+
+import (
+	"os"
+
+	"github.com/quickfeed/quickfeed/internal/rand"
+)
+
+// AuthSecret returns the secret used to sign JWT tokens.
```

Comment:
```go
// AuthSecret returns the JWT signing secret obtained from the QUICKFEED_AUTH_SECRET
// environment variable, or a random secret if unset.
```
I'm not sure we want to return a random secret if it is unset, since we should require that `QUICKFEED_AUTH_SECRET` is set when we start the server or that we set it via the `-secret` flag (it gets saved and loaded in `main.go`.)

However, I don't like the idea of panicking if it is unset either. @JosteinLindhom what do you think?